integration/testomitempty.graphql

```diff
@@ -1,3 +1,25 @@
 type RemoteModelWithOmitempty {
     newDesc: String
 }
+
+type DummyUserWithRemoteNamedBasics {
+    newString: String!
+    newInt: Int!
+    newInt32: Int!
+    newInt64: Int!
+    newBool: Boolean!
+    newFloat64: Float!
+    newID: ID!
+
+    ### BELOW ARE NOT SUPPORTED AS BASIC - MUST CREATE SCALAR WITH MARSHALFUNC AND UNMARSHARLFUNC ###
```

Comment:
Is this comment still correct?

Reply:
Yes, for some reason, the ones below still has the message `is incompatible with`, and I am not sure why. I could spend some time to investigate if you want.

Reply:
Sorry, I did a quick test and the error message is not `..incompatible with..` but `Undefined type uint64` coming from `gqlparser`.